test/features/caseUpdate.feature

```diff
@@ -0,0 +1,19 @@
+@functional
+Feature: Set of scenario's covering update functionality with a second user on a created case data
+
+  Background:
+    Given I have logged in as 'auto.test.cnp+fe2@gmail.com'
```

Comment:
This user needs to be moved to the env variable CCD_CASEWORKER_AUTOTEST_FE2_EMAIL.